tpcx_bb/queries/q10/tpcx_bb_query_10.py

```diff
@@ -103,8 +103,9 @@ def main(client, config):
 
     # These files come from the official TPCx-BB kit
     # We extracted them from bigbenchqueriesmr.jar
-    neg_sent_df = load_sentiment_words("negativeSentiment.txt", "NEG")
-    pos_sent_df = load_sentiment_words("positiveSentiment.txt", "POS")
+    sentiment_dir = "/".join(config["data_dir"].split("/")[:-3] + ["sentiment_files"])
```

Comment:
This is fine for now, but we should be careful. It looks like it's assuming a certain level of nesting in the directory structure which may not be the case unless we direct people to use this structure.

Reply:
Should also eventually switch this to path joins in #15